server/src/app.js

```diff
@@ -3,13 +3,15 @@ const cors = require('cors')
 const swaggerUi = require('swagger-ui-express')
 const swaggerSpecs = require('./swaggerConfig')
 const HealthRoute = require('./routes/HealthRoute')
+const UserRoute = require('./routes/UserRoute')
 
 const app = express()
 
 app.use(express.json())
 app.use(cors())
 
 app.use('/api/v1/health', HealthRoute)
+app.use('/users', UserRoute)
```

Comment:
NON:BLOCKING - So we're not going to versioned our api? i.e. /api/v1/users?

Reply:
No, was not aware of versioning best practices. Will update the code to include versioning: /api/v1/users

Reply:
Caching: https://github.com/codesydney/classified-ads-app-for-good/issues/28
Pagination: https://github.com/codesydney/classified-ads-app-for-good/issues/29